### PR TITLE
Add state-info ellipsis + fix height

### PR DESCRIPTION
**Pull request by spacegaier**
## Proposed change

1. Add ellipsis to state name
2. Remove fixed height to achieve the more compact design of name and relative-time we had before

=> Does it perhaps even make sense to use a hover-maquee for the name or is the intended solution to use the existing double click on the dialog header to go to a wider dialog if you need the whole name? Or add a tooltip with the whole name? 

**Before:**
image

**After:**
image

## Type of change

- [ ] Dependency upgrade
- [X] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue or discussion:
- Link to documentation pull request:

## Checklist

- [X] The code change is tested and works locally.
- [X] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

**Comment by Misiu** [comment]
@spacegaier I like the idea to make dialog wider by double-clicking on the dialog header (like here: https://github.com/home-assistant/frontend/pull/7492).
The whole name will be visible, but there is one extra benefit: the graph will be wider and more readable.
Just my 2¢.

**Comment by spacegaier** [comment]
@Misiu That option already exists today. My wording above probably was a bit misleading.

**Review by bramkragten — APPROVED** [review]

File: src/components/entity/state-info.ts [hunk 1]
```diff
@@ -120,7 +120,8 @@ class StateInfo extends LitElement {
       .name {
         @apply --paper-font-common-nowrap;
```
Comment (bramkragten) [inline src/components/entity/state-info.ts:121]: I don't think this works, can we check what was in here and see what else is missing?

Comment (bramkragten) [reply src/components/entity/state-info.ts:121]: (Same for line 97) cc: @iantrich

Comment (iantrich) [reply src/components/entity/state-info.ts:121]: That is what was there, but I think you're right in that `@apply‘ does not work with Lit. That's a Polymer thing, I believe. So I totally missed that part of the conversion.

**Pull request merged**